### ISSUE TITLE
Doesn't work if exporting on same line as definition

### ISSUE THREAD

**Issue by garyking** [bryanrsmith/eslint-plugin-sort-class-members#47]
If I have:

```
class Test {
  constructor() {
    this.test = true;
  }

  static test(arg) {
    return arg;
  }
}

module.exports = Test;
```

Then I will get an error, that `static` should be before `constructor`. But if I export on the same line that I define the class, by changing the first line to `module.exports = class Test {`, then this plugin doesn't show any errors.
